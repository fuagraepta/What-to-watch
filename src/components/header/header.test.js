import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header.jsx';

const mockComponent = <div></div>;

describe(`The component is rendered correctly`, () => {
  it(`Header correctly renders with transferred mock children`, () => {
    const header = renderer
    .create(
        <Header>
          {mockComponent}
        </Header>
    )
    .toJSON();

    expect(header).toMatchSnapshot();
  });
});
