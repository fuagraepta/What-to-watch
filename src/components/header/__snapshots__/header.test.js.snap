// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Header correctly renders with transferred mock children 1`] = `
<header
  className="page-header movie-card__head"
>
  <div />
</header>
`;
